server/pom.xml

```diff
@@ -109,6 +109,10 @@
             <artifactId>testcontainers</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
if the correlation id is not needed when server starts, then the dependency will not be needed.